services/ui-src/src/components/tables/SortableDrawerReportPageTable.tsx

```diff
@@ -0,0 +1,153 @@
+import { Button, Image } from "@chakra-ui/react";
+import { useMemo } from "react";
+// components
+import { generateColumns, SortableTable } from "components";
+import deleteIcon from "assets/icons/icon_cancel_x_circle.png";
+
+export const SortableDrawerReportPageTable = ({ entities }: Props) => {
+  const actualData = useMemo(() => {
+    const result = entities.map((entity: any, index: number) => {
```

Comment:
nit: could do `return entities.map((` and get rid of the `result` return at the bottom